peachjam/templates/peachjam/_subject-area-cases-indices.html

```diff
@@ -0,0 +1,10 @@
+{% load peachjam i18n %}
```

Comment:
I'm not sure what's going on with this filename. :) The taxonomy tree is very generic, how about `_taxonomy_tree_facet.html` as a filename, and "Subject areas" as a heading?

Reply:
done